### DISCUSSION TITLE
clist not recognized

### DISCUSSION THREAD

**Discussion by shadowtempest91** [lptstr/winfetch#193] [General]
Greetings. Since a few days I'm encountering this bug when i run winfetch:

![Cattura](https://github.com/lptstr/winfetch/assets/94082702/2f3b2777-a66f-43b8-9fd8-d2b38a3358dc)

It's in italian language, as you see. The translations goes this way:

```
Term 'clist' not recognized as a name of cmdlet, function, executable program or script file. Check the ortography of the name or verify that the path is included and correct, then retry.
In C:\Users\rapto\scoop\apps\winfetch\current\winfetch.ps1:875 car:24
+         $chocopkg = (& clist -l)[-1].Split(' ')[0] - 1
+                        ~~~~~llllll
    + CategoryInfo          : ObjectNotFound: (clist:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
```

I previously installed it via choco; I uninstalled it and installed it again on scoop. The issue is exactly the same, with the difference that in both cases it was counting only the packages installed either via choco or scoop, as you see in the screenshot. Previously, when it worked correctly, it was counting both choco and scoop packages.

I suppose that some upgrades might have broken it. I have no clue about which ones.

**Comment by Dev-Negandhi** [comment]
I have been encountering the same issue. This code seems to appear in line 875 of the script

I guess it is some problem with the information for chocolatey

**Comment by rashil2000** [comment]
See #189
